Formula/g/gcc@13.rb

```diff
@@ -107,6 +84,11 @@ def install
       # System headers may not be in /usr/include
       sdk = MacOS.sdk_path_if_needed
       args << "--with-sysroot=#{sdk}" if sdk
+
+      # Avoid this semi-random failure:
+      # "Error: Failed changing install name"
+      # "Updated load commands do not fit in the header"
+      make_args = %w[BOOT_LDFLAGS=-Wl,-headerpad_max_install_names]
```

Comment:
Is this preferable to the `--with-boot-ldflags` argument of `configure`?

Reply:
We used the make variable before, I think because otherwise the default value of `--with-boot-ldflags` is overwritten. I'm just choosing the same as other existing gcc versioned formulas, for consistency.